<?xml version="1.0" encoding="UTF-8"?><Posts><Post><PostId>23319806</PostId><Url>http://stackoverflow.com/questions/23319806</Url><Stack><Type>CommandFailed</Type><Message>Command '['/usr/local/bin/cinder', '--os-username', 'xxx', '--os-tenant-name', 'xxx', '--os-password', 'xxx', '--os-auth-url', 'http://xxxx:5000/v2.0', '--endpoint-type', 'publicURL', 'availability-zone-list']' returned non-zero exit status 1</Message><Frame><File>/root/tempest.bak/tempest/cli/simple_read_only/test_cinder.py</File><Line>107</Line><Method>test_cinder_availability_zone_list</Method></Frame><Frame><File>/root/tempest.bak/tempest/cli/__init__.py</File><Line>84</Line><Method>cinder</Method></Frame><Frame><File>/root/tempest.bak/tempest/cli/__init__.py</File><Line>109</Line><Method>cmd_with_auth</Method></Frame><Frame><File>/root/tempest.bak/tempest/cli/__init__.py</File><Line>129</Line><Method>cmd</Method></Frame></Stack></Post></Posts>